aiperf/common/config/user_config.py

```
@@ -225,6 +225,43 @@ def _count_dataset_entries(self) -> int:
+        ),
+        DisableCLI(reason="Automatically parsed from --gpu-telemetry argument"),
+    ] = GPUTelemetryMode.SUMMARY
+
+    gpu_telemetry_urls: Annotated[
+        list[str],
+        Field(
+            default_factory=list,
+            description="Parsed GPU telemetry DCGM endpoint URLs",
+        ),
+        DisableCLI(reason="Automatically parsed from --gpu-telemetry argument"),
+    ] = []
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)